index.bs

```diff
@@ -2497,12 +2503,17 @@ the credential. It is intended primarily for [RPS] that wish to tightly control
     the available authenticators match a provided AAGUID, the client MUST select an authenticator from among the available
     authenticators to generate the credential.
 
+: Client data
+:: Returns the JSON value `true` to indicate to the RP that the extension was acted upon
+
```

Comment:
Should specify both key and value. Ideally in such a way that the server does not need to do complicated matching of request and response. May be better here just to include the supplied client argument in ClientData, with the same key.